### ISSUE TITLE
Sql -> h2

### ISSUE THREAD

**Issue by Bennyboy1695** [greatman/craftconomy3#21]
hi i was wondering if there was any easy ways that you knew of converting the sql data for your plugin over to h2 data instead. i know theres documentation out there but as your the plugin developer i thought you would be bound to know of an easy and simple conversion tool

**Comment by 9190281** [comment]
I'm not a developer o.o

**Comment by greatman** [comment]
You want Mysql => H2 ?

**Comment by Bennyboy1695** [comment]
Yeah thats correct and was wondering if you knew of any ways that will make us not lose any data and have minimal downtime while the data is changed over :P

**Comment by greatman** [comment]
Porting it is rather easy, H2 is simply a SQL database engine very similar to MySQL in terms of SQL commands support. There's nothing in cc3 atm doing the MySQL => H2 conversion since nobody asked for it. People usually do the other way around.

**Comment by Bennyboy1695** [comment]
we have been having sql issues with cc3 lately and another plugin and was going to try moving it to a h2 database to see if it solves the issue or at least temporarily fixes them otherwise we wouldn't be moving over

**Comment by greatman** [comment]
What kind of issues? Probably just a question of configuration / optimization.
